gf-webapp-admin/src/component/registration/Registration.module.scss

```diff
@@ -37,6 +37,7 @@
   font-size: 1.1rem;
```

Comment:
Need to check fonts in all projects. Sometimes its rem, sometimes px, and put it in consts

Reply:
we shoul create  global variable for font sizes

Reply:
BTW: please specify global variables in gf-ui-lib (not in each module separate)

Reply:
Done